.docs/.sphinx/spellingcheck.yaml

```diff
@@ -30,6 +30,11 @@ matrix:
             - a.contributor
             - span.pre
             - .sig-object
+            - '#module-apt em'
+            - '#module-apt strong'
+            - '#module-apt a.reference'
+            - a.reference[title^="apt."]
+            - a[href*="#apt"]
```

Comment:
Yeah, this is pretty nasty! @dwilding, you mentioned that the new spell checker is significantly better or more forgiving. Would that help us here?